network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -0,0 +1,25 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+	<div class="tw-row tw-py-4">
+		{% for block in self.cards %}
+          {% with card=block count=self.cards|length %}
+            {% image card.image fill-350x197 as img %}
+            <div class="tw-px-4 small:tw-w-full {% if count > 2 %} large:tw-w-1/2 medium:tw-w-1/2 {% else %} medium:tw-w-1/2 {% endif %} tw-mb-5">
```

Comment:
Can you add some more context around the `if` conditions here? Shouldn't `medium:tw-w-1/2` be enough, since we ask the editor to add at least 2 cards? Also, isn't ` large:tw-w-1/2 medium:tw-w-1/2` the same as `medium:tw-w-1/2` (all screens above medium use half width)?

Reply:
Thanks, you are right, now we are actually enforcing the rule I've remove the unnecessary classes https://github.com/mozilla/foundation.mozilla.org/pull/9434/commits/1c2728e4b4f7799edcfd494877cdee3c403a8b47